sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlignAssignmentsSuite.scala

```diff
@@ -36,7 +36,7 @@ import org.apache.spark.sql.execution.datasources.v2.V2SessionCatalog
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.types.{BooleanType, IntegerType, StructType}
 
-class AlignAssignmentsSuite extends AnalysisTest {
+abstract class AlignAssignmentsSuite extends AnalysisTest {
```

Comment:
nit: if this is a base suite, probably add a `Base` suffix in the name, to be consistent with `RowLevelOperationSuiteBase`

Reply:
Agree, updated.